### PR TITLE
Adding Citadel RP dogborgs(Alternative Pupdozer model/Servicehound)

### PR DESCRIPTION
**Pull request by Holdelta**
Pure port of dogborgs, may add additional features later.

Additions:
-Vale Pupdozer model
-Servicehounds, four different options

### CODE REVIEW

File: code/global_vr.dm [hunk 1]
```diff
@@ -4,12 +4,13 @@
 	robot_module_types += "Janihound"
 	robot_module_types += "Sci-borg"
 	robot_module_types += "Pupdozer"
+	robot_module_types += "Servicehound"
```
Comment (Repede) [inline code/global_vr.dm]: Add //YW Addition in front of line please

File: code/global_vr.dm [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
 	return 1
 
 var/list/shell_module_types = list(
 	"Standard", "Engineering", "Surgeon",  "Crisis",
 	"Miner", "Janitor", "Service", "Clerical",
-	"Medihound", "Janihound"
+	"Medihound", "Janihound", "Servicehound"
```
Comment (Repede) [inline code/global_vr.dm]: Add //YW Addition in front of line please

File: code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm [hunk 3]
```diff
@@ -43,6 +43,7 @@
 	robot_modules["Janihound"] = /obj/item/weapon/robot_module/robot/scrubpup
 	robot_modules["Sci-borg"] = /obj/item/weapon/robot_module/robot/science
 	robot_modules["Pupdozer"] = /obj/item/weapon/robot_module/robot/engiedog
+	robot_modules["Servicehound"] = /obj/item/weapon/robot_module/robot/servicehound
```
Comment (Repede) [inline code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm]: Add //YW Addition in front of line please

File: code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm [hunk 4]
```diff
@@ -479,7 +480,8 @@
 	name = "Construction Hound module"
 	sprites = list(
 					"Pupdozer" = "pupdozer",
-					"Borgi" = "borgi-eng"
+					"Borgi" = "borgi-eng",
+					"V2 Engidog" = "thottbot"
```
Comment (Repede) [inline code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm]: Add //YW Addition in front of line please

File: code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm [hunk 5]
```diff
@@ -603,6 +605,78 @@
...
+	R.pixel_x 	 = -16
+	R.old_x 	 = -16
+	R.default_pixel_x = -16
+	R.dogborg = TRUE
+	R.wideborg = TRUE
+	R.verbs |= /mob/living/silicon/robot/proc/ex_reserve_refill
+	R.verbs |= /mob/living/silicon/robot/proc/robot_mount
+	R.verbs |= /mob/living/silicon/robot/proc/rest_style
+	..()
+
```
Comment (Repede) [inline code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm]: Could this be added to a new file called station_yw in the same folder? Or have it delimited with //YW Additions start and //YW Additions end.

Comment (Holdelta) [reply code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm]: Okay, I believe I have resolved all documenting issues, sorry about that, didn't know about it.

**Comment by Repede** [comment]
Seems Travis broke with this...
Would you be able to close this PR, and open a new one please?

**Pull request closed without merging**